records.py

```diff
@@ -165,6 +165,11 @@ def dataset(self):
         # Create a new Tablib Dataset.
         data = tablib.Dataset()
 
+        # If the RecordCollection is empty, just return the empty set
+        # Check number of rows by typecasting to list
+        if len(list(self)) == 0:
```

Comment:
I was aiming for readability since both are equally expensive. Looking at
my code, one can easily judge that it is checking if the list is empty
rather than if it is a None object or something else.
On Apr 12, 2016 10:24 PM, "Timo Furrer" <notifications@github.com> wrote:

> In records.py
> <https://github.com/kennethreitz/records/pull/65#discussion_r59484664>:
>
> > @@ -165,6 +165,11 @@ def dataset(self):
> >          # Create a new Tablib Dataset.
> >          data = tablib.Dataset()
> >
> > +        # If the RecordCollection is empty, just return the empty set
> > +        # Check number of rows by typecasting to list
> > +        if len(list(self)) == 0:
>
> You can actually do:
>
> if not list(self):
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kennethreitz/records/pull/65/files/98ac44d450d7970f5289d520199e238f372dc2d8#r59484664>
>